src/components/resource-section/add-evaluation/AddEvaluationNoAuth.js

```diff
@@ -0,0 +1,44 @@
+import { InputTextarea } from 'primereact/inputtextarea';
+import { Rating } from 'primereact/rating';
+import { Card } from 'primereact/card';
+import { Button } from 'primereact/button';
+import useLoginDialog from '@hooks/useLoginDialog';
+import { Tooltip } from 'primereact/tooltip';
+
+const AddEvaluationNoAuth = () => {
+  const loginDialog = useLoginDialog();
+  return (
+    <div
+      className="cardEvaluation"
```

Comment:
Not sure if is correct to use the className out of the style file.

Reply:
Same as above